tests/test_jquery_installed.py

```diff
@@ -4,11 +4,18 @@
 
 import pytest
 import sphinx
-from sphinx.testing.path import path
 from sphinx.testing.util import SphinxTestApp
+from packaging.version import parse
 
 from sphinxcontrib.jquery import _FILES, _ROOT_DIR  # NoQA
 
+last_with_legacy = parse("7.1.2")
+if parse(sphinx.__version__) > last_with_legacy:
```

Comment:
Just a suggestion: you can use `sphinx.version_info` which is already a tuple, rather than parsing a string.

```python3
if sphinx.version_info >= (7, 2):
    # New Sphinx
else:
    # Old Sphinx
```

Reply:
Thanks for the suggestion!